neurodsp/sim/aperiodic.py

```diff
@@ -238,6 +239,82 @@ def sim_powerlaw(n_seconds, fs, exponent=-2.0, f_range=None, **filter_kwargs):
 
     return sig
 
+@normalize
+def sim_fgn(n_seconds, fs, hurst=0.5):
+    """Simulate a fractional gaussian noise time series with specified Hurst exponent.
+
+    Parameters
+    ----------
+    n_seconds : float
+        Simulation time, in seconds.
+    fs : float
+        Sampling rate of simulated signal, in Hz.
+    hurst : float, optional, default: -0.5
+        Desired Hurst exponent.
+
+    Returns
+    -------
+    sig: 1d array
+        Fractional gaussian noise time series with the desired Hurst exponent.
+
+    Notes
+    -----
+    The relationship between the power law exponent beta and the Hurst exponent for fractional
+    gaussian noise is hurst = (beta + 1)/2.
+
+    Examples
+    --------
+    Simulate fractional gaussian noise with an exponent of -0.5 (white noise):
+
+    >>> sig = sim_fgn(n_seconds=1, fs=500, hurst=0.5)
+    """
+
+    # Construct the autocovariance function.
```

Comment:
this is pure nitpicking - but by convention we don't add periods to inline comments (unless there are multiple sentences)